app/build.gradle

```diff
@@ -81,21 +81,33 @@ android {
         sourceCompatibility 1.8
         targetCompatibility 1.8
     }
+    kotlinOptions {
+        jvmTarget = '1.8'
+        freeCompilerArgs = ['-Xjvm-default=enable',
+                            "-Xuse-experimental=kotlinx.coroutines.ExperimentalCoroutinesApi",
+                            "-Xuse-experimental=kotlinx.coroutines.ObsoleteCoroutinesApi"]
+    }
     androidExtensions {
         experimental = true
     }
 }
 
+apollo {
+    // instruct the compiler to generate Kotlin models
+    generateKotlinModels.set(true)
+}
+
 repositories {
     maven { url = uri("https://dl.bintray.com/kodein-framework/Kodein-DI") }
 }
 
 dependencies {
     implementation fileTree(dir: 'libs', include: ['*.jar'])
     implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk8:$versions.kotlin"
+    implementation 'com.google.code.gson:gson:2.8.6'
```

Comment:
Probably can be removed

Reply:
gson using in com.flatstack.android.util.storage.Storage